src/components/Alphabet/AlphabetContainer.js

```diff
@@ -5,12 +5,12 @@ import AlphabetPresentation from 'components/Alphabet/AlphabetPresentation'
 import AlphabetPlaceholder from 'components/Alphabet/AlphabetPlaceholder'
 import AlphabetData from 'components/Alphabet/AlphabetData'
 import AlphabetPresentationWidget from 'components/Alphabet/AlphabetPresentationWidget'
-import Loading from 'components/Loading'
+import LoadOrError from 'components/LoadOrError'
 
 function AlphabetContainer({ widgetView = false, kids = null }) {
   const {
     characters,
-    isLoading,
+    characterQueryReturn,
```

Comment:
Minor: elsewhere in the code we call objects like this responses rather than queryReturns. Would it make sense to stick to one name?

Reply:
I did consider this exact question but as the LoadOrError component relies specifically on properties included in the return from a useQuery hook I was trying to highlight that. Do you think response is the better way to go?

Reply:
Ohhhh such a good line of thinking. Most of our data hooks are useQuery under the hood, but some are useInfiniteQuery, right? Maybe we could do something like queryResponse and infiniteQueryResponse, and leave the misc "response" objects from other functions as they are? This is making me want to switch to Typescript